tf-psa-crypto/include/psa/crypto.h

```diff
@@ -4918,11 +4918,13 @@ uint32_t psa_generate_key_iop_get_num_ops(psa_generate_key_iop_t *operation);
  *                              aborted by calling \c
  *                              psa_generate_key_iop_abort().
  *
+ * \note                        Only asymmetric keys are supported. (See \p attributes.)
```

Comment:
Public keys can't be generated.
```suggestion
 * \note                        Only asymmetric key pairs are supported. (See \p attributes.)
```